src/get_maintainer_wrapper.sh

```diff
@@ -47,40 +48,168 @@ function print_files_authors()
     done
 }
 
+# Returns true if given dir is a kernel tree root. Argument is the dir.
+function is_kernel_root()
+{
+  local -r DIR="$@"
+  # The following files are some of the files expected to be at a linux
+  # tree root and not expected to change. Their presence (or abscense)
+  # is used to tell if a directory is a linux tree root or not. (They
+  # are the same ones used by get_maintainer.pl)
+  if [[ -f "${DIR}/COPYING" &&
+  		-f "${DIR}/CREDITS" &&
+		-f "${DIR}/Kbuild" &&
+		-e "${DIR}/MAINTAINERS" &&
```

Comment:
Why you use '-e' instead of '-f' here?

Reply:
Oh, nice question. The get_maintainer.pl script uses this exactly same tests to check for a kernel root. So I used '-e' instead of '-f' to keep this function consistent with get_maintainer.pl

Reply:
Oh, nice question. The get_maintainer.pl script uses this exactly same tests to check for a kernel root. So I used '-e' instead of '-f' to keep this function consistent with get_maintainer.pl.

Reply:
Oh, nice question. The get_maintainer.pl script uses this exactly same tests to check for a kernel root. So I used '-e' instead of '-f' to keep this function consistent with get_maintainer.pl.